test/solvers/functional/default_functionals_test.py

```diff
@@ -0,0 +1,319 @@
+# Copyright 2014-2016 The ODL development group
+#
+# This file is part of ODL.
+#
+# ODL is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ODL is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with ODL.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Test for the default functionals."""
+
+# External
+import numpy as np
+import pytest
+
+# Internal
+import odl
+from odl.util.testutils import all_almost_equal, almost_equal, noise_element
+
+# Places for the accepted error when comparing results
+PLACES = 8
+
+
+def test_L1_norm():
```

Comment:
These tests should be more extensive. I suggest adding fixtures for `space` so that we can verify that the computation works for whatever cases we are looking for (at least the `fn_impls` spaces with both `rn` and `uniform_discr`, but i wouldnt put a non-uniform discr out of the question since I guess we'll find bugs there.)